.nuspec/Xamarin.Forms.props

```diff
@@ -15,6 +10,6 @@
 	The actual item groups are in a separate conditionally-imported file as they use constructs that
 	are not compatible with older MSBuild versions.
 	-->
-	<Import Project="$(MSBuildThisFileDirectory)Xamarin.Forms.DefaultItems.props" Condition="'$(IsSdkStyle)' == 'true'" />
+	<Import Project="$(MSBuildThisFileDirectory)Xamarin.Forms.DefaultItems.props" Condition="'$(EnableDefaultItems)' == 'true'" />
```

Comment:
It looks like the EnableDefaultItems property hasn't been set by this point.

If I change the condition to 
```
<Import Project="$(MSBuildThisFileDirectory)Xamarin.Forms.DefaultItems.props" Condition="'$(EnableDefaultItems)' == ''" />
```

Then it pulls the props file in and it compiles